Podcast/SeriesDetailViewController.swift

```diff
@@ -131,7 +129,7 @@ class SeriesDetailViewController: ViewController, SeriesDetailHeaderViewDelegate
                 self.episodeTableView.finishInfiniteScroll()
                 self.continueInfiniteScroll = false
             }
-            self.series!.episodes = self.series!.episodes + episodes
+            self.episodes = self.episodes + episodes
```

Comment:
will the cache take care of making sure there are no duplicates returned? If not, we should keep around a set

Reply:
Pretty sure cache takes care of this

Reply:
It takes care of it somewhat. Also I didn't remove a set, there was never a set in this class.